destination_sdk_v2.proto

```diff
@@ -0,0 +1,181 @@
+syntax = "proto3";
+option optimize_for = SPEED;
+option java_multiple_files = true;
+option go_package = "fivetran.com/fivetran_sdk/v2";
+package fivetran_sdk.v2;
+
+import "google/protobuf/timestamp.proto";
+import "common_v2.proto";
+
+// Fivetran (grpc client) <> DestinationConnector (grpc server)
+service DestinationConnector {
+    rpc ConfigurationForm (ConfigurationFormRequest) returns (ConfigurationFormResponse) {}
+    rpc Capabilities(CapabilitiesRequest) returns (CapabilitiesResponse) {}
+    rpc Test (TestRequest) returns (TestResponse) {}
+    rpc DescribeTable (DescribeTableRequest) returns (DescribeTableResponse) {}
+    rpc CreateTable(CreateTableRequest) returns (CreateTableResponse) {}
+    rpc AlterTable(AlterTableRequest) returns (AlterTableResponse) {}
+    rpc Truncate(TruncateRequest) returns (TruncateResponse) {}
+    rpc WriteBatch (WriteBatchRequest) returns (WriteBatchResponse) {}
+}
+
+message CapabilitiesRequest {}
+
+message CapabilitiesResponse {
+    BatchFileFormat batch_file_format = 1;
+    repeated DataTypeMappingEntry data_type_mappings = 2;
+    bool supports_history_mode = 3;
+}
```

Comment:
The order of these fields are different in `engineering` repo. Please figure out which is correct and update the incorrect one.